src/components/review-card-ratings-only.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Grid, Card, Progress, Rating } from 'semantic-ui-react';
+
+const RatingsCard = (props) => {
+  const { reviewTitle, usefulProgress, workloadProgress, difficultyProgress,
+    enjoymentProgress, reviewDate, overallRating } = props;
+
+  return (
+
+    <div style={{ display: 'block', margin: '20px' }}>
+      <Card fluid raised>
+        <Card.Content>
+          {/* <div style={{ margin: '100' }}>
+            <Progress value={enjoymentProgress} total='5' progress='ratio' size='small' color='blue'/>
+          </div> */}
+
+          <Grid>
+            <Grid.Row className='review-row-ratings-only'>
+
+              <Grid.Column width={16}>
+
+                <div style={{ margin: '0px 260px 0px 0' }}></div>
+                <span style={{ float: 'right' }}>{reviewDate}</span>
+
+                <Card.Header className="header-test">
+                  <span>
+                    <h3 className='header-test' >{reviewTitle}</h3>
+
+
+                  </span>
+
+
+                </Card.Header>
+
+
+                <Card.Meta style={{ margin: '5px 0' }}>
+                    Overall: <Rating icon='star' defaultRating={overallRating} maxRating={5} disabled />
+
+                </Card.Meta>
+
+                <div className='bar-chart-rows' >
+                      Usefulness
```

Comment:
This is a bit nit picky but here we have a **label** for a Progress bar, so best to use

<label>Usefulness</label>
<Progress .... etc. />

I think you put the progress in a div so that it would appear on a new line, but its better semantically to use above structure, and then you can apply styling to <label> to make new line break if needed

Reply:
check out https://www.w3schools.com/tags/tag_label.asp